pkgs/tools/compression/flips/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, gtk3, pkg-config }:
+
+stdenv.mkDerivation {
+  pname = "flips";
+  version = "1.31";
+
+  src = fetchFromGitHub {
+    owner = "Alcaro";
+    repo = "Flips";
+    rev = "24e6c77eb8d1ed6d0eb6da400c67ea229eca2278";
+    sha256 = "17b9mbpj59w6n5nin9g98rz4q9nz42m5wdir53jaw8cglz9xmsyp";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ gtk3 ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp flips $out/bin/flips
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Please add `platforms`.

Reply:
Done.

Reply:
If you're able to fix the `Makefile` and submit patch upstream that is one option, or you can simply adjust `platforms`. I'm good either way.

Reply:
Restricted list of platforms to Linux, because I don't use macOS, so it would be tricky to debug the issue.